server_admin/topics/vault.adoc

```diff
@@ -88,7 +82,7 @@ and master secret of the keystore that was created by Elytron. An example of the
 </spi>
 ----
 
-If the underlying keystore has a format other than `JCEKS`, this format has to be informed using the `keyStoreType`:
+If the underlying keystore has a format different than `JCEKS`, you must specify this format by using the `keyStoreType`:
```

Comment:
I prefer "different from" but we could debate it  https://www.thesaurus.com/e/grammar/different-from-vs-different-than/

Also, delete "you must"